src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -418,11 +419,16 @@ def _check_pid_from_log_file(self, pid, abort_on_warning, leak_wf, show_memleaks
                     if show:
                         # Allow WARNING or ERROR messages, but anything higher
                         # like assert should trigger a failure.
-                        if line.level < cart_logparse.LOG_LEVELS['ERR']:
-                            show_line(line, 'HIGH', 'error in strict mode')
-                        else:
-                            show_line(line, 'NORMAL', 'warning in strict mode')
-                        warnings_mode = True
+                        show_this = True
+                        for ignores in self.skip_suffixes:
+                            if line.get_msg().endswith(ignores):
+                                show_this = False
+                        if show_this:
```

Comment:
FYI - for a one-liner that also early-exits (`map` returns a generator)
```suggestion
                        if not any(map(line.get_msg().endswith, self.skip_suffixes)):
```

Reply:
This PR is going to conflict with https://github.com/daos-stack/daos/pull/12112 whatever I do here so I'll leave this for now and fix up once that has landed.